project-base/app/composer.json

```diff
@@ -157,7 +157,7 @@
             "php app/downloadPhing.php --phing-version=3.0.0": "script",
             "php phing clean": "script",
             "shopsys:domains-urls:configure": "symfony-cmd",
-            "ckeditor:install --clear=skip --release=full --tag=4.5.11": "symfony-cmd"
```

Comment:
this version requires paid license afaik

Reply:
But previous used version has been detected with security issue, and recommended this versions. How can I check license usage?

Reply:
![obrazek](https://github.com/user-attachments/assets/700ae0e2-d0db-46a1-aa38-5d1be6d3cf4a)

Console: This CKEditor 4.5.11 version is not secure. Consider upgrading to the latest one, 4.25.0-lts: https://ckeditor.com/ckeditor-4-support/

Reply:
Ok according to slack discussion this change will be reverted.